### PR TITLE
GH-1984 Propagate DBL Click support

### PR DESCRIPTION
**Pull request by GordonSmith**
Fixes GH-1984

Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@jbrundage @jchambers-ln @mlzummo Please Review

**Comment by GordonSmith** [comment]
@jchambers-ln I would be inclined to push this back to 1.14.8?

**Comment by jchambers-ln** [comment]
Agree that looks to be a larger one-- I'll still review.

File: src/chart/MultiChart.js [hunk 1]
```diff
@@ -143,7 +143,10 @@
         if (arguments.length) {
             var context = this;
             _.click = function (row, column, selected) {
-                context.click(row, column, selected);
+                context.click.apply(context, arguments);
```
Comment (jchambers-ln) [inline src/chart/MultiChart.js]: Is param selected used now, is it needed?

File: src/common/Icon.js [hunk 2]
```diff
@@ -54,13 +54,24 @@
...
+            })
+        ;
     };
 
     Icon.prototype.click = function (domNode) {
         console.log("Clicked the icon");
     };
 
+    Icon.prototype.dblclick = function (domNode) {
+        console.log("Double clicked the icon");
```
Comment (jchambers-ln) [inline src/common/Icon.js]: should that log statement be controlled so it can be disabled?

File: src/map/GMapPinLine.js [hunk 3]
```diff
@@ -16,6 +16,9 @@
             .on("click", function (row, col, sel) {
                 context.click(context.rowToObj(row.ext.origRow), "", sel);
             })
+            .on("dblclick", function (row, col, sel) {
```
Comment (jchambers-ln) [inline src/map/GMapPinLine.js]: Not consistent naming -- some of the functions in this pull have (row, col, sel) some have (row, column, selected)

File: src/chart/MultiChart.js (hunk 1 above)
Comment (GordonSmith) [inline src/chart/MultiChart.js]: ```context.click.apply(context, arguments);``` FYI this means call the "click" event on a base class passing in the current argument list - so row, column + selected all get passed to the base click event.

File: src/map/GMapPinLine.js (hunk 3 above)
Comment (GordonSmith) [inline src/map/GMapPinLine.js]: True.

**Pull request merged**